source_code/run.py

```diff
@@ -0,0 +1,113 @@
+import json
+from pathlib import Path
+from typing import List, Tuple
+
+import click
+
+from git_repo_extract.commits_info import get_commits_info_floored
+from git_repo_extract.star_track import get_stargazer_info, get_top_repos
+from git_repo_extract.repo_ops import operate_temporary_repo
+from code_parsing.code_handle import parallelize_extraction
+from utils import file_writer, parallel_function, split_into_batches, write_down_content
+
+
+@click.group()
+def cli():
+    """
+    A group of cli methods
+    """
+    pass
+
+
+@cli.command()
+@click.option("--repos_file_path", default=Path.cwd().parent / "cloned_repos" / "selected_repos.txt", type=click.Path())
```

Comment:
create constant that will help you to avoid duplication of code & possible errors during modification
```
DEAFULT_DIR = Path.cwd().parent / "cloned_repos" 
```